server/main.example.go

```diff
@@ -0,0 +1,26 @@
+package server
```

Comment:
If so, I would move all the source files except main.example.go into coniksServer subpackage.